### PR TITLE
Fix control bar button label styling. Fix screenshare button hover and checked styling.

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What

1. Fix background color and label of screenshare button
  * **NOTE: MOVED STYLING FROM COMPOSITES TO COMPONENT**
2. Fix our control bar buttons so that the label can be styled correctly
  * **Simplified our label code** - removed our custom label rendering logic, please review

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/2633102

# How Tested
* Locally tested:
  * image
  * image
  * image
  * image

* Will check UI tests once they complete.
* Tested keyboard nav 
  * @alcail please comment any extra accessibility tests to check

### CODE REVIEW

File: packages/react-components/src/components/ScreenShareButton.tsx [hunk 1]
```diff
@@ -67,3 +71,18 @@ export const ScreenShareButton = (props: ScreenShareButtonProps): JSX.Element =>
     />
   );
 };
+
+const screenshareButtonStyles = (theme: Theme): IButtonStyles => ({
```
Comment (JamesBurnside) [inline packages/react-components/src/components/ScreenShareButton.tsx:75]: NOTE: Now in components and not composites

File: packages/react-components/src/components/ControlBarButton.tsx [hunk 2]
```diff
@@ -154,9 +133,11 @@ export const ControlBarButton = (props: ControlBarButtonProps): JSX.Element => {
       <DefaultButton
         {...props}
         styles={componentStyles}
-        onRenderText={props.showLabel ? props.onRenderText ?? DefaultRenderText : undefined}
+        onRenderText={props.showLabel && props.onRenderText ? props.onRenderText : undefined}
         onRenderIcon={props.onRenderIcon ?? DefaultRenderIcon}
-      />
+      >
+        {props.showLabel ? labelText : <></>}
```
Comment (JamesBurnside) [inline packages/react-components/src/components/ControlBarButton.tsx:139]: NOTE: No longer using a custom render function (so get the styling from props correctly)

**Review by alcail — APPROVED** [review]

**Comment by alcail** [comment]
just make sure that when focused on (navigated to through keyboard) the outline is indeed white and Narrator indeed says "Stop sharing button", but code seems great so should be no issue

File: change/@internal-react-composites-90f8fc03-9c00-426e-a0d9-44028a8259ed.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Moved code",
```
Comment (anjulgarg) [inline change/@internal-react-composites-90f8fc03-9c00-426e-a0d9-44028a8259ed.json:3]: Can we be specific in this comment about which code was removed?
Helps when we go over the combined changelog later.

File: packages/react-components/src/components/ScreenShareButton.tsx [hunk 4]
```diff
@@ -67,3 +71,18 @@ export const ScreenShareButton = (props: ScreenShareButtonProps): JSX.Element =>
...
+  rootChecked: {
+    background: theme.palette.themePrimary,
+    color: DefaultPalette.white,
```
Comment (anjulgarg) [inline packages/react-components/src/components/ScreenShareButton.tsx:78]: curious about the usage of `DefaultPalette` for color instead of `theme`?

**Review by anjulgarg — APPROVED** [review]

File: change/@internal-react-composites-90f8fc03-9c00-426e-a0d9-44028a8259ed.json (hunk 3 above)
Comment (JamesBurnside) [reply change/@internal-react-composites-90f8fc03-9c00-426e-a0d9-44028a8259ed.json:3]: This is a no-op for this package (nothing changes from the perspective of the composites package) so type is 'none'. When the changelog is generated it omits these

File: packages/react-components/src/components/ScreenShareButton.tsx (hunk 4 above)
Comment (JamesBurnside) [reply packages/react-components/src/components/ScreenShareButton.tsx:78]: I left this as is (this code was just moved from one file to another) but also curious on this too if anyone knows why default palette is used

File: change/@internal-react-composites-90f8fc03-9c00-426e-a0d9-44028a8259ed.json (hunk 3 above)
Comment (anjulgarg) [reply change/@internal-react-composites-90f8fc03-9c00-426e-a0d9-44028a8259ed.json:3]: Oh cool. Didn't know that.

File: packages/react-components/src/components/ScreenShareButton.tsx (hunk 4 above)
Comment (anjulgarg) [reply packages/react-components/src/components/ScreenShareButton.tsx:78]: I think @PorterNan worked on this originally.

**Comment by JamesBurnside** [comment]
> Awesome. I just checked to make sure that when focused on (navigated to through keyboard) the outline is indeed white and Narrator indeed says "Stop sharing button", and perfect :)

Confirmed both, thanks!

**Pull request merged**